api/tests/test_environment.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2015 Planet Labs, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+import simplejson as json
+from datalake_api import v0
+
+
+def test_health(client, monkeypatch, tmpdir):
+    uri = '/v0/environment/'
+    p = tmpdir.join("test_sha.txt")
+    p.write('\nfake_version\n\n')
+    monkeypatch.setattr(v0, "get_sha_path", lambda: p.strpath)
```

Comment:
Hm. Not juiced about mocking random internal code paths. Can we just make the test write the version file? The change won't persist in the container. It's kludgey. But less so IMO than mocking.

Reply:
Adjusted test such that it doesn't check the explicit version value, and thus does not use mocking, or overwrite the existing version file.